<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SloyWidget</class>
 <widget class="QWidget" name="SloyWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>831</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>sloy1</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="postGr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="postLabel">
        <property name="text">
         <string>Условный номер станции:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="postSpin"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="altLabel">
        <property name="text">
         <string>Высота станции над уровнем моря, м</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="altSpin">
        <property name="minimum">
         <number>-100</number>
        </property>
        <property name="maximum">
         <number>9999</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="stationWidgetGr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="StationWidget" name="stationWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Отбор:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="centerLabel">
        <property name="text">
         <string>Центр:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QRadioButton" name="stationRadio">
        <property name="text">
         <string>по станции</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QComboBox" name="centerCombo"/>
      </item>
      <item row="2" column="1">
       <widget class="QRadioButton" name="analyseRadio">
        <property name="text">
         <string>по данным анализа</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDateTimeEdit" name="dateEdit">
        <property name="displayFormat">
         <string>HH:mm  &quot;dd&quot; MMMM yyyy (UTC)</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="timeSpec">
         <enum>Qt::UTC</enum>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Срок:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0" colspan="3">
       <widget class="ZondWidget" name="zond">
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StationWidget</class>
   <extends>QWidget</extends>
   <header location="global">meteo/commons/ui/custom/stationwidget.h</header>
  </customwidget>
  <customwidget>
   <class>ZondWidget</class>
   <extends>QTableView</extends>
   <header location="global">meteo/commons/ui/custom/zondwidget.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>postSpin</tabstop>
  <tabstop>altSpin</tabstop>
  <tabstop>stationRadio</tabstop>
  <tabstop>analyseRadio</tabstop>
  <tabstop>centerCombo</tabstop>
  <tabstop>dateEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>analyseRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>centerCombo</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>192</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>192</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>analyseRadio</sender>
   <signal>toggled(bool)</signal>
   <receiver>centerLabel</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>47</x>
     <y>368</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
